src/Composer/Satis/Builder/DownloadsBuilder.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * This file is part of Satis.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *     Nils Adermann <naderman@naderman.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Composer\Satis\Builder;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Helper\HelperSet;
+use Composer\Factory;
+use Composer\IO\ConsoleIO;
+
+/**
+ * Builds the archives of the repository.
+ *
+ * @author James Hautot <james@rezo.net>
+ */
+class DownloadsBuilder extends Builder implements BuilderInterface
```

Comment:
Downloads is rather generic. Can you find something more specific to the purpose of this class?